packer/ansible/image-build.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - import_playbook: /infrastructure/ansible/setup.yml
 - import_playbook: /infrastructure/ansible/monitoring.yml
+  vars:
+    datadog_api_key: DATADOG_API_KEY
```

Comment:
Do you mean the literal `DATADOG_API_KEY` or the value that the var `DATADOG_API_KEY` has in the environment as set by import-secrets.sh?
I am not sure what is the intended value, and whether this is a specific Ansible syntax.

Reply:
It's literal. The idea is that this key is set to some "random" string while building the image. We don't want secrets build-in into the machine images.

Then during the boostrap it's set to a working secret api key. it cannot be set to falsy value unfortunately to be more clear. I changed it a bit d76efa2, I hope it's better now.